opencti-worker/src/worker.py

```diff
@@ -702,9 +721,10 @@ def start(self) -> None:
                             self.opencti_ssl_verify,
                             self.opencti_json_logging,
                         )
+                        self.consumer_threads[push_queue].name = push_queue
                         self.consumer_threads[push_queue].start()
                     # Listen for webhook message
-                    if connector["config"].get("listen_callback_uri") is not None:
+                    if "connector_user" in connector and connector["config"].get("listen_callback_uri") is not None:
```

Comment:
did not test much this part but I ran into issues with external import connectors. looking at the code, it seems to be wrong to use listen_callback_uri as a condition to create an ApiConsumer because it is always set up by the connector helper (https://github.com/OpenCTI-Platform/client-python/blob/073609f05b968455e9ecb905612bdedb523c14dd/pycti/connector/opencti_connector_helper.py#L1105)

Reply:
resolving with https://github.com/OpenCTI-Platform/client-python/pull/867